_includes/navbar.html

```diff
@@ -118,6 +118,10 @@
             <li><a href="/faq/what-is-the-build-field.html">What is the <code>build</code> object in the ontology metadata?</a></li>
             <li><a href="/docs/OntologyStatus.html">What does it mean for an ontology to be flagged as 'inactive', 'orphaned' or 'obsolete'?</a></li>
             <li role="separator" class="divider"></li>
+            <li class="dropdown-header"><i>OBO Dashboard</i></li>
+            <li><a href="/faq/dashboard.html">What is the OBO Dashboard?</a></li>
+            <li><a href="/faq/dashboard.html">How can I assess my ontology using the OBO Dashboard?</a></li>
```

Comment:
Same link, yea?

Reply:
yes. same link. both answers are on that same page. i thought about adding an anchor but the page isn't very long.